src/main.py

```diff
@@ -23,6 +22,9 @@
 
 controllers: Dict[int, BustController] = {}
 
+# Cached image to use for next bust
+loaded_image: Optional[str] = None
```

Comment:
Hmm, if ever wanted to make concurrent busts a thing this would need to be scoped to a server. But for now I think it's fine.

Reply:
Yes, though they're already but possible because the attachments directory will remove anything listed from another server on `!list`. So we can change this if we ever want to, but it's fine for now imo